src/yunohost/app.py

```diff
@@ -1031,6 +1031,23 @@ def _get_setting(settings, name):
     for domain in domains:
         skipped_urls.extend([domain + '/yunohost/admin', domain + '/yunohost/api'])
 
+    # Authorize ACME challenge url if a domain seems configured for it...
+    for domain in domains:
+
+        # Check ACME challenge file is present in nginx conf
+        nginx_acme_challenge_conf_file = "/etc/nginx/conf.d/%s.d/000-acmechallenge.conf" % domain
+
+        if not os.path.isfile(nginx_acme_challenge_conf_file):
+            continue
+
+        # Check the file contains the ACME challenge uri
+        if not '/.well-known/acme-challenge' in open(nginx_acme_challenge_conf_file).read():
+            continue
+
+        # If so, then authorize the ACME challenge uri to unprotected regex
+        unprotected_regex.append(domain + "/%.well%-known/acme%-challenge/.*$")
+
```

Comment:
Seems to work ! Added in dc731c3. We need to validate that we're okay with having this skipped regex for all domains enabled by default though. I don't think it poses any security threat...